### PR TITLE
Sprints/careyprice

### PR DESCRIPTION
**Pull request by pierluc-ssoucy**

### CODE REVIEW

File: src/Solution/Solution/Entities/opc_complaint/Entity.xml [hunk 1]
```diff
@@ -1362,6 +1362,68 @@
...
+          <optionset Name="opc_complaint_opc_hasotherallegation">
+            <OptionSetType>bit</OptionSetType>
+            <IntroducedVersion>1.0.358</IntroducedVersion>
+            <IsCustomizable>1</IsCustomizable>
+            <ExternalTypeName></ExternalTypeName>
+            <displaynames>
+              <displayname description="Has Other Allegation" languagecode="1033" />
+            </displaynames>
+            <Descriptions>
+              <Description description="Determines if the Complaint is associated with allegations other than TL or Access" languagecode="1033" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_complaint/Entity.xml:1405]: missing french

File: src/Solution/Solution/Entities/opc_complaint/Entity.xml [hunk 2]
```diff
@@ -1362,6 +1362,68 @@
...
+                  <label description="No" languagecode="1033" />
+                </labels>
+              </option>
+            </options>
+          </optionset>
+          <displaynames>
+            <displayname description="Has Other Allegation" languagecode="1033" />
+          </displaynames>
+          <Descriptions>
+            <Description description="Determines if the Complaint is associated with allegations other than TL or Access" languagecode="1033" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_complaint/Entity.xml:1424]: missing french

File: src/Solution/Solution/Entities/opc_complaint/FormXml/main/{b8bed16c-fe15-4801-ae58-2636ceecce77}.xml [hunk 3]
```diff
@@ -211,6 +211,23 @@
...
+                          <label description="Has Access Allegation" languagecode="1033" />
+                          <label description="A une allégation d’accès" languagecode="1036" />
+                        </labels>
+                        <control id="opc_hasaccessallegation" classid="{67FAC785-CD58-4F9F-ABB3-4B7DDC6ED5ED}" datafieldname="opc_hasaccessallegation" disabled="false" />
+                      </cell>
+                    </row>
+                    <row>
+                      <cell id="{19ca0e30-3f00-47d0-adaa-04fb6910e572}" locklevel="0" colspan="1" rowspan="1" visible="false">
+                        <labels>
+                          <label description="Has Other Allegation" languagecode="1033" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_complaint/FormXml/main/{b8bed16c-fe15-4801-ae58-2636ceecce77}.xml:226]: missing french

File: src/Solution/Solution/Other/Relationships/Account.xml [hunk 4]
```diff
@@ -8,6 +8,7 @@
     <ReferencedEntityName>Account</ReferencedEntityName>
     <CascadeAssign>NoCascade</CascadeAssign>
     <CascadeDelete>RemoveLink</CascadeDelete>
+    <CascadeArchive>NoCascade</CascadeArchive>
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Other/Relationships/Account.xml:11]: @mitchkirady this looks like a new feature. heard of it?

File: src/Solution/Solution/Workflows/Allegation-CreateUpdate-SetHasTLAccessAllegation-C72D1463-3D95-EA11-A812-000D3AF46B49.json [hunk 5]
```diff
@@ -37,12 +37,9 @@
               "apiId": "/providers/Microsoft.PowerApps/apis/shared_commondataserviceforapps"
             },
             "parameters": {
-              "subscriptionRequest/message": 4,
+              "subscriptionRequest/message": 7,
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Workflows/Allegation-CreateUpdate-SetHasTLAccessAllegation-C72D1463-3D95-EA11-A812-000D3AF46B49.json:40]: rename the flow according to the new behavior

File: src/Solution/Solution/Workflows/IndividualComplaint-3E0E9F7E-183E-4B3E-8B89-E668EB6B48C8.xaml.data.xml [hunk 6]
```diff
@@ -74,8 +74,14 @@
       <label languagecode="1033" description="Reason to close" />
     </steplabels>
     <steplabels id="{9d805c29-feb7-4c5f-a45c-c13d1e8c9765}">
+      <label languagecode="1033" description="Has Access Allegation" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Workflows/IndividualComplaint-3E0E9F7E-183E-4B3E-8B89-E668EB6B48C8.xaml.data.xml:77]: missing french

File: src/Solution/Solution/Other/Relationships/Account.xml (hunk 4 above)
Comment (mitchkirady) [reply src/Solution/Solution/Other/Relationships/Account.xml:11]: I haven't seen this, but will keep an eye on it.

File: src/WebResources/typings/xrmdefinitelytyped/_internal/Enum/processstage_operationkind.d.ts [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  JsonToText = 473330016,
+  XmlToJson = 473330017,
+  XmlToText = 473330018,
+  Geofence = 473330019,
+  ODataOpenApiConnection = 473330020,
+  IndexOf = 473330021,
+  Substring = 473330022,
+  VirtualAgent = 473330023,
+  FormatNumber = 473330024,
+}
```
Comment (pierluc-ssoucy) [inline src/WebResources/typings/xrmdefinitelytyped/_internal/Enum/processstage_operationkind.d.ts:27]: @mitchkirady do this ring any bell?

Comment (mitchkirady) [reply src/WebResources/typings/xrmdefinitelytyped/_internal/Enum/processstage_operationkind.d.ts:27]: not at all :(

**Pull request merged**